deploy/manifests/prod/us-east-2/tenant/storetheindex/instances/inga/pvc_data.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  labels:
+    app: indexer-single
+    app.kubernetes.io/managed-by: kustomization
+    app.kubernetes.io/part-of: storetheindex
+  name: data
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Ti
```

Comment:
ago disk on dev is `0.1`% utilised. I think we can reduce this more and expand as needed. Let's start with with 10Gi

Reply:
Done